src/sql/data_misc.sql

```diff
@@ -1115,7 +1115,7 @@ INSERT INTO `geodesic_classifieds_ad_configuration` (`number_of_photos_in_detail
+('http://www.MyListingsSite.com/index.php', 'index.php', '', 'http://www.MyListingsSite.com/aff.php', 0, 'GeoClassifieds Enterprise', 'Nothing bigger or better', '', 760, 'center', 0, '', 0, 'shorterthan20', 0, 0, 'http://www.MyListingsSite.com/register.php', '', 0, 1, 1, 1, 1, 0, 1, 1, 0, 0, 1, 1, 0, 0, 0, 3, 4, 1, 0, 1, 'ffff80', 'ffffff', '000066', 'images/misc/photo_icon2.gif', 1, 1, 3, 20, 0, 40, 50, 0, 'images%2Fmisc%2Fquestion_mark.gif', 'images/misc/attn_gtr_sold2.gif', 1, 0, 0, 0, 0, 1, 1, 1, 0, 0, 30, 1, 0.00, 'paypal@geodesicsolutions.org', 0.0000, '', '$', 'US', 30, 1, 150, 150, 6, 0, 0, 0, 29, 0, 15, 5, 15, 0, 0, 0, 0, 0, 2, 10, 0, 1, 0, 'M j, Y, G:i:s', 5, 150, 150, 0, 1, 1, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, '', '', 0, 0, 120, 'images/misc/new_ad.gif', '', 1, 0, 1, 0, 1000, 'images/misc/nophoto.gif', 119, 0, 0, 0, 0, 0, 0, 0, 0, '', 3, 5, 4, 'utf-8', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 1, 1, '', 'images/misc/buynow2.gif', 'images/misc/resmet.gif', 1, 1, 'images/misc/nores.gif', 0, 0, 0, 0, 0, 1, 0, 'site_off.htm', 0, 0, 0, 'Optional Field 1', 'Optional Field 2', 'Optional Field 3', 'Optional Field 4', 'Optional Field 5', 'Optional Field 6', 'Optional Field 7', 'Optional Field 8', 'Optional Field 9', 'Optional Field 10', 'Optional Field 11', 'Optional Field 12', 'Optional Field 13', 'Optional Field 14', 'VIN', 'Optional Field 16', 'Optional Field 17', 'Optional Field 18', 'Optional Field 19', 'Shipping & Handling', -1, 0, 1, 0, 'http://www.MyListingsSite.com/stores.php', 'M j, Y', 0);
```

Comment:
Instead of `paypal@geodesicsolutions.org`, lets make it something that is clear it would never work - `paypal@example.com`.

`example.com` is a good one for documentation and the like, it is a [reserved domain specifically for use in documentation](https://www.iana.org/domains/reserved) and the like.  And it could never be used for a real paypal address since you need to verify your email when signing up with paypal.

Reply:
> did you do a test run through the setup/

Other than installing site from scratch, is there a better way to test it? Are you mainly concerned that the SQL executes w/o error? If so, I could just source the sql file on the database.